src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -121,7 +121,7 @@ export default class BusinessLookupFetch extends Mixins(CommonMixin, SearchMixin
         // set conversionType and entityTypeCd because there's only one alteration type for it
         if (this.isBenBusiness) {
           this.setConversionType(NrRequestTypeCodes.CONVERT_CORP)
-          this.setEntityTypeCd(EntityTypes.BC)
+          this.setEntityTypeCd(EntityTypes.CR)
```

Comment:
But what about this line of code that says BECR is for a Benefit Company?

https://github.com/bcgov/namex/blob/5e6cee7267f812782daad65fcddf34093b185d17/api/namex/services/lookup/name_request_filing_actions.py#L116

Reply:
The corresponding namex code change: https://github.com/bcgov/namex/pull/1468/files

Reply:
I am confused. What does BECR now mean? Doesn't it still apply to Benefit Companies (entity type BC in Namerequest; corp type BEN in LEAR)?

cc: @argush3

Reply:
My understanding, BECR means alteration from BEN to BE Limited Company. Both entity_type_cd and legal_type are for the company type after alteration. entity_type_cd is used in NR, while legal_type is used in LEAR. The original company type is not recorded.

Reply:
On the other hand, your code change in Namex API is consistent with the definition in LEAR:
https://github.com/bcgov/lear/blob/4a3a32603e92c3c03287ac8e7e7b79fccd14d2a3/legal-api/src/legal_api/resources/v1/nr_type_map.py#L177

So this might work... Can you test it before merging the code changed?

Reply:
Travis tested and it worked. LEAR works with legal_type='BC'.

Reply:
![image](https://github.com/bcgov/namerequest/assets/54647458/815ea2c7-8744-40fd-9db8-30b4052a23cc)

Reply:
in PROD, entity_type_cd should be 'CR' instead of 'BC' for BECR.

Reply:
OK, since this has been tested then I'm OK with it. Thanks.

Reply:
Thank you!